pkgs/by-name/ni/nixos-render-docs-redirects/src/nixos_render_docs_redirects/__init__.py

```diff
@@ -0,0 +1,86 @@
+import json
+import argparse
+from pathlib import Path
+
+
+def add_new_identifier(redirects: dict[str, list[str]], identifier: str, path: str) -> dict[str, list[str]]:
```

Comment:
```suggestion
def add_identifier(redirects: dict[str, list[str]], identifier: str, path: str) -> dict[str, list[str]]:
```

Typically one only ever adds new things